frontend/src/scenes/insights/InsightDisplayConfig.tsx

```diff
@@ -154,32 +154,33 @@ export function InsightDisplayConfig({ filters, activeView, disableTable }: Insi
                 )}
             </div>
             <div className="flex items-center space-x-4 flex-wrap my-2">
-                {activeView === InsightType.TRENDS && (
-                    <ConfigFilter>
-                        <span>{axisLabel(filters.display)}</span>
-                        <LemonSelect
-                            value={filters.aggregation_axis_format || 'numeric'}
-                            onChange={(value) => {
-                                if (value) {
-                                    setFilters({ ...filters, aggregation_axis_format: value })
-                                }
-                            }}
-                            bordered
-                            dropdownPlacement={'bottom-end'}
-                            dropdownMatchSelectWidth={false}
-                            disabled={!canFormatAxis(filters.display)}
-                            data-attr="chart-aggregation-axis-format"
-                            options={aggregationAxisFormatSelectOptions}
-                            type={'stealth'}
-                            size={'small'}
-                        />
-                    </ConfigFilter>
-                )}
                 {showChartFilter(activeView) && (
-                    <ConfigFilter>
-                        <span>Chart type</span>
-                        <ChartFilter filters={filters} disabled={filters.insight === InsightType.LIFECYCLE} />
-                    </ConfigFilter>
+                    <>
+                        {activeView === InsightType.TRENDS && (
```

Comment:
To me it was a bit clearer what are the conditions for the unit to appear when this was a level higher, because now knowledge of `showChartFilter()` (which should really be `showChartTypeSelector()`) is required, and `showChartFilter()` seems kind of redundant here because of the `=== InsightType.TRENDS` anyway. But that's a nit.

Reply:
Yeah, it's verging on tricky...

The chart filter is shown when trends or stickiness but the units only on trends. So was trying to express that these two things go together but not in every case 🤔